app/lib/frontend/templates/publisher.dart

```diff
@@ -162,6 +165,14 @@ String renderPublisherAdminPage(Publisher publisher) {
   );
 }
 
+String _renderDetailHeader(Publisher publisher) {
+  final shortCreated = shortDateFormat.format(publisher.created);
+  return renderDetailHeader(
+    title: publisher.publisherId,
+    metadataHtml: htmlEscape.convert('Publisher registered on $shortCreated.'),
```

Comment:
It could also say: `Created: $shortCreated`? But I'm not sure it's a better option...

Maybe `Publisher created: $shortCreated`?

Reply:
My issue with the "created" is the following: when I visit the publisher page, and I see that a publisher was created a few months ago, I may infer that all of their packages are younger than that. Using "registered" may shift that perception a bit, because it implies (at least for me) that the packages are not necessarily younger than that date.

Reply:
ah, over time I don't think that'll matter much... but good point.